src/runtime_src/core/common/smi_default.h

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2025 Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
+#include <string>
+
+namespace xrt_core::smi_default {
+// This is the default behavior when no shim is loaded.
```

Comment:
Please explain in comments what this is used for and why it is in coreutil and not part of xrt-smi itself.  E.g. who is using this API.    If the API needs to be in coreutil, then it should be merged with xrt_core::smi in smi.cpp.